astrosat/admin.py

```diff
@@ -188,3 +269,20 @@ def get_tags_for_list_display(self, obj):
         return get_clickable_m2m_list_display(DatabaseLogTag, obj.tags.all())
 
     get_tags_for_list_display.short_description = "tags"
+
+    def export_as_csv(self, request, queryset):
+
+        fields_to_export = ["id", "uuid", "level", "created", "message", "tags"]
```

Comment:
I suspect all they will really want is `level`, `created` and `message`, even `level` is probably unnecessary as `message` will contain error message. Not looking for any change though, just a comment.